README.md

```diff
@@ -1,13 +1,25 @@
 # Project Auth
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The assignment was to create a backend including RESTful API with endpoints that allows the user to register, sign-in and reach a page only when authenticated, otherwise getting sent back to the login/register screen.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+Tech used:
+> React
+> Redux
+> React Router
+> bcrypt (password encryption combining hashing and salting)
+> crypto (access token generation)
+> MongoDB Cloud Atlas (database hosting)
+> Heroku (server hosting)
+
+Endpoints:
+/secret   from which the secret message (requiring authentication) is fetched
+/signup   authenticates a new user and returns status code 400 if user with said username already exists
+/signin   authenticates a user if said user is found as registered and the given password matches the stored one after hashing and salting has been applied
+
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://epic-bassi-8dc187.netlify.app/
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```

![image](https://user-images.githubusercontent.com/50926159/120907224-1ff14e00-c660-11eb-8f24-d188aa585391.png)